### PR TITLE
Update JSON Schema and Context

### PR DESCRIPTION
**Pull request by OR13**
- fix lint errors
- added json schema for service, proof, etc..
- add context and documentation for hosting `usage`

### CODE REVIEW

**Comment by OR13** [comment]
I specifically did not try and fix all warnings, but there were a few files which I needed to save changes to, and now that prettier is installed correctly, that caused some formatting changes.

File: packages/element-lib/.eslintrc [hunk 1]
```diff
@@ -1,34 +1,3 @@
...
-    "node/no-unsupported-features/es-syntax": "off",
-    "node/no-unsupported-features/es-builtins": "off",
-    "node/no-unpublished-require": [
-      "error",
-      {
-        "allowModules": ["jest-express"]
-      }
-    ]
-  }
+  "extends": ["@transmute/eslint-config/eslint.config.react.json"]
```
Comment (gjgd) [inline packages/element-lib/.eslintrc]: why config.react for element-lib?

Comment (OR13) [reply packages/element-lib/.eslintrc]: good point, this should not be react

File: docs/contexts/sidetree/sidetree-v0.0.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Sidetree Context
+
+- [Latest JSON-LD Context](./sidetree-v0.0.jsonld)
+
+### Terminology
+
+<h4 id="usage"><a href="#usage">usage</a></h4>
```
Comment (gjgd) [inline docs/contexts/sidetree/sidetree-v0.0.md]: this link points to itself

File: packages/element-lib/package.json [hunk 3]
```diff
@@ -69,13 +69,12 @@
...
-    "eslint": "^5.10.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^22.1.2",
-    "eslint-plugin-node": "^8.0.0",
-    "eslint-plugin-security": "^1.4.0",
+    "eslint": "^6.8.0",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-jest": "^23.2.0",
+    "eslint-plugin-node": "^11.0.0",
```
Comment (gjgd) [inline packages/element-lib/package.json]: Why are we removing `eslint-config-airbnb-base` and `eslint-plugin-security` ? they are both needed by eslint.config.node.json.

The eslint console in vscode can show what plugins are missing

File: docs/contexts/sidetree/sidetree-v0.0.md (hunk 2 above)
Comment (OR13) [reply docs/contexts/sidetree/sidetree-v0.0.md]: its supposed to its for building an anchor to the definition.

File: packages/element-lib/package.json (hunk 3 above)
Comment (OR13) [reply packages/element-lib/package.json]: hmm, these should be added, I'm surprised to build passes without them.

File: docs/contexts/sidetree/sidetree-v0.0.md (hunk 2 above)
Comment (OR13) [reply docs/contexts/sidetree/sidetree-v0.0.md]: for example:

http://element-did.com/contexts/sidetree/sidetree-v0.0.md#usage

this is part of jsonld.

File: packages/element-lib/package.json (hunk 3 above)
Comment (OR13) [reply packages/element-lib/package.json]: I don't see errors when editing element-lib locally or when running lint on the build server, this seems to imply that these are not required, but I added them anyway.

**Pull request merged**